src/markdown_renderers.js

```diff
@@ -115,8 +115,10 @@ export const generateGitRepoUrlPrefix = (repoUrl, branchName, href) => {
 */
 export const customLinkRenderer = (href, text, repoUrl, branchName) => {
     // Ensure that anchors have lowercase href
-    if (href.startsWith('#')) {
+    // Return Apify links without special rels
+    if (href.startsWith('#') || href.includes('apify.com')) {
```

Comment:
Maybe add a comment to the function what is happening for the Apify domain links